api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -20,5 +20,18 @@
     <include file="liquibase-update-to-2.1.xml"/>
     <include file="liquibase-update-to-2.2.xml"/>
     <include file="liquibase-update-to-2.3.xml"/>
-
+<changeSet id="Trunk-5411:Add dose limit unit field to Drug" author="gitacliff">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="drug" columnName="dose_limit_unit"/>				
+			</not>
+		</preConditions>
+		<comment>Adding column dose limit unit in table drug</comment>
+        <addColumn tableName="drug">
+            <column name="dose_limit_unit" type="int">
+            	<constraints nullable="false"/>
```

Comment:
Are you sure that this column should not allow nulls?

Reply:
@dkayiwa my thinking on this is that since this column is going to will represent the units of the existing fields of maximumDailyDose and minimumDailyDose of a given drug and a do believe every drug must these fields catered for which necessitates the doselimitunit not to all nulls